src/trace/Kconfig

```diff
@@ -31,4 +24,249 @@ config TRACEM
 	help
 	  Sending all traces by mailbox additionally.
 
+config TRACEV
+	bool "Trace verbose"
+	depends on TRACE
+	default n
+	help
+	  Enabling verbose traces.
+
+config TRACE_VERBOSE_IRQ
```

Comment:
Submenu?

Reply:
@paulstelian97 
I personally don't feel the need for submenu, especially, since these _depend_ on TRACEV, they don't clutter the menu, until user decides to enable TRACEV, at which point these options appear instantly, allowing user to pick trace classes to enable.

Reply:
Fair enough then.

Reply:
@akloniex Can you surround them with `if TRACEV` & `endif # TRACEV` ?

Reply:
If applies `depends` and there is no need to repeat it on every entry

Reply:
done

Reply:
done